oscpy/server.py

```diff
@@ -327,7 +325,7 @@ def send_bundle(
         send_bundle(messages, ip_address, port, sock=sock, safer=safer)
 
     def answer(
-        self, address=None, values=None, bundle=None, timetag=None, safer=False
+        self, address=None, values=None, bundle=None, timetag=None, safer=False, port=None
```

Comment:
please split line after to stick to pep8 (before "safer", because the comma hit the 80 mark).

Reply:
Urgh yes - sorry, this was done in haste. Will update.

Reply:
No worries, i'm seeing i was far from perfect on the pep8 side, fixing the other ones on my side :)